registry.md

```diff
@@ -7,35 +27,88 @@ This registry may be expanded with new RFCs.
-| Ethereum | Ethereum network       | TBD       | `ethereum` | `network`        |                    | The network on which to operate |
-|          |                        |           |            |                  | `regtest`          | Local dev network               |
-|          |                        |           |            |                  | `ropsten`          | Ropsten testnet (network id 3)  |
-|          |                        |           |            |                  | `mainnet`          | Ethereum mainnet (network id 1) |
+The following is a list of possible values a `Ledger` type header can take:
 
+| Value      | Reference                            | Description                            |
+|:-----------|--------------------------------------|----------------------------------------|
+| `bitcoin`  | [RFC-004](./RFC-004-SWAP-Bitcoin.md) | The Bitcoin Core family of blockchains |
+| `ethereum` | [RFC-006](./RFC-006-SWAP-Ethereum.md)| The Ethereum family of blockchains     |
+
+
+And the possible parameters they each may have:
+
+### `bitcoin` Parameters
+
+| Parameter | Value Type                                  | Description                              |
+|:----------|---------------------------------------------|------------------------------------------|
+| `network` | see [Bitcoin Networks](#bitcoin-networks) | The particular blockchain netowrk to use |
+
+#### Bitcoin Networks
+
+| Value     | Description                          |
+|:----------|:-------------------------------------|
+| `regtest` | Private Bitcoin Core regtest network |
+| `testnet` | Bitcoin Core testnet                 |
+| `mainnet` | Bitcoin Core mainnet                 |
 
-[RFC003](./RFC-003-SWAP-basic.md#identity) requires that each ledger has an associated identity:
 
-| Ledger   | Identity Name | JSON Encoding            | Reference | Description                                                                       |
-|:----     |:-------      |:-------------            |:--------- | --------------------------------------------------------------------------------- |
-| Bitcoin  | `pubkeyhash`  | `hex-encoded-bytes (20)` | TBD       | The result of applying SHA256 and then RIPEMD160 to a user's SECP256k1 public key |
-| Ethereum | `address`     | Ethereum address        | TBD       | An Ethereum address                                        |
+### `ethereum` Parameters
 
+| Parameter | Value Type                                    | Description                              |
+|:----------|-----------------------------------------------|------------------------------------------|
+| `network` | see [Ethereum Networks](#ethereum-networks) | The particular blockchain netowrk to use |
+
+
+#### Ethereum Networks
+
+| Value     | Description                      | Network Id |
+|:----------|:---------------------------------|------------|
+| `regtest` | Private Ethereum regtest network | N/A        |
```

Comment:
Should we mention here that the parity dev network is an example for this?

Reply:
Hmm, 
I we should just call this thing "dev" for both bitcoin and ethereum -- In other words both nodes are in the same environment and connected to the same development node so there's no need to convey anything about it in the swap message. Then there's no need to give examples.

Also, I think no one in Ethereum calls anything "regtest".

Reply:
Maybe we shouldn't maintain this list in the RFC at all but reference to somewhere else? 
For example, what if this `dev` mode id changes?

Reply:
Including @bonomat relevant comment from the other PR:
> For our setup it is `17` --> this is important as with [comit-network/comit-rs#817](https://github.com/comit-network/comit-rs/pull/817) the network will be retrieved from the ledger btsieve is connected to.
> 
> Suggested change 
> | `regtest` | Private Ethereum regtest network | N/A        |
> | `regtest` | Private Ethereum regtest network | 17 (for [parity dev-chain](https://wiki.parity.io/Private-development-chain))        |

We don't care about the network id for the dev network for specification of this protocol. It's always `N/A`. We can just assume "dev" whatever the nodes interpret it to mean in their test environment. If they get it wrong then that's a problem with the test environment.

I think we can delete the network id column in the RFC for now as it just seems to be confusing. Maybe we can take an approach where we pass it as another parameter in a revision to this RFC.

Reply:
Removed the network id and added a reminder for myself to mention this int he tracking issue.